hooks/boot/20-certs_update.rb

```diff
@@ -18,6 +18,14 @@
   :default => false
 )
 app_option(
+  '--certs-reset',
+  :flag,
+  "This option will reset any custom certificates and use the self-signed CA " \
+  "instead. Note that any clients will need to be updated with the latest " \
+  "katello-ca-consumer RPM.",
```

Comment:
1) Does this cause a need to regen and re-do Proxies as well?
 2) If I reset my server back to the default CA, can I leave my Proxies on the custom certs?